### DISCUSSION TITLE
fail to install foundry on windows

### DISCUSSION THREAD

**Discussion by xyl2004** [foundry-rs/foundry#8990] [Help] [answered]
The download of Foundry failed. Here is a screenshot of the error message. What should I do?
![屏幕截图 2024-09-30 205256](https://github.com/user-attachments/assets/29f29b4c-539b-45f6-a878-5764e33d3b1f)

**Comment by Niferu** [comment]
Not sure, but I highly recommend using Ubuntu on windows and install foundry via ubuntu terminal, it is much faster and simpler to install without any issues. I have tested both solutions and trust me Ubuntu is the best option here

**Comment by zerosnacks** [comment, marked as answer]
This could be related to a firewall setting blocking access

You can find the relevant scripts used here: https://github.com/foundry-rs/foundry/tree/master/foundryup

Alternatively download and install from source
